src/Tizen.NUI/src/internal/Application/NUICoreBackend.cs

```diff
@@ -80,15 +80,10 @@ public NUICoreBackend(string stylesheet, NUIApplication.WindowMode windowMode, S
         /// The constructor with stylesheet, window mode, window size, window position and default window type.
         /// This will be hidden as inhouse API. Because it is only for internal IME window.
         /// </summary>
-        [EditorBrowsable(EditorBrowsableState.Never)]
-        public NUICoreBackend(string stylesheet, NUIApplication.WindowMode windowMode, Size2D windowSize, Position2D windowPosition, WindowType type)
+        public NUICoreBackend(string stylesheet, NUIApplication.WindowMode windowMode, WindowType type)
```

Comment:
I don't think this `public NUICoreBackend(...)` is not for the ACR (this will not be used by user), so please maintain the `[EditorBrowsable(EditorBrowsableState.Never)]' to become a hidden api.

Reply:
I see, this file is in internal folder,  I think  [EditorBrowsable(EditorBrowsableState.Never)] is not needed.
I will add that.